ngReact.js

```diff
@@ -9,17 +9,17 @@
 (function (root, factory) {
   if (typeof module !== 'undefined' && module.exports) {
     // CommonJS
-    module.exports = factory(require('react'), require('angular'));
+    module.exports = factory(require('react'), require('react-dom'), require('angular'));
   } else if (typeof define === 'function' && define.amd) {
     // AMD
-    define(['react', 'angular'], function (react, angular) {
-      return (root.ngReact = factory(react, angular));
+    define(['react', 'react', 'angular'], function (react, reactDOM, angular) {
```

Comment:
As i seen in https://fb.me/react-dom-0.14.0-rc1.js, they used `define(['react'])`, so we should still use `'react'`, but I never used AMD at before, maybe I'm wrong.